Ensembl_to_NCBI_AM_061324.py

```diff
@@ -0,0 +1,74 @@
+#Ensembl to NCBI Accession Codes for homo sapiens. 
+#This is just my first attempt at this kind of project and am only aiming to print out the ID's for NCBI. 
+#Example: ENSG00000012048 
+import requests
+import re
+from Bio import Entrez
+import urllib.parse
+import json
+import xml.etree.ElementTree as ET
+from constants import summary_url
+
+if __name__=="__main__":
+    Entrez.email = input("In compliance with NCBI policy, please input your email:") 
+
+    e_id = input("Please type in a valid Ensembl ID:").strip() 
+#For future scripts, will try to have this be a list or data frame column to loop through 
+# but let's just focus on getting this to work with one for now. - 061324
```

Comment:
We should move this into a config.py file that will store all of these variables. The current py file could then reference that file. This way one file is doing one action.